### ISSUE TITLE
Clarification on getting podcast urls

### ISSUE THREAD

**Issue by Kartikeya99** [gpodder/gpodder#961]
I was reading the code of gpodder-core and podcast-parser and both of them require URLs of the podcasts to work. I was wondering from where does gpodder get the list of the URLs/RSS feed of all the podcasts? 
Does it have some kind of a database where it stores this list? If so, from where can I generate that list? #

**Comment by tpikonen** [comment]
I'm not sure if I understood your question, but in general the podcast RSS/Atom URLs are provided by the user by their own means (web search etc.)

There is some content discovery functionality in the menu bar under 'Subscriptions->Discover new podcasts', which mostly points to the (currently very slow) gpodder.net website.

**Comment by Kartikeya99** [comment]
Yes, on the gpodder.net website, when you search for something or when you use the open API, it returns results from its own database. So my question was how does it populate its own database.

**Comment by auouymous** [comment]
I believe there is a form for feed owners to submit their podcast.
